tests/Integration/Console/CommandEventsTest.php

```diff
@@ -171,6 +172,12 @@ protected function writeArtisanScript()
     {
         $path = base_path('artisan');
 
+        while ($this->fs->exists("$path.lock")) {
+            //
```

Comment:
Wouldn't it be better to add a sleep in these while loops?
```suggestion
        while ($this->fs->exists("$path.lock")) {
            usleep(250 * 1000);
```

Reply:
I will consider it, thanks!